### DISCUSSION TITLE
Drop submodules

### DISCUSSION THREAD

**Discussion by sdumetz** [Smithsonian/dpo-voyager#283] [Ideas]
I already professed my hate for submodules on more than one occasion.

As the `ff-*` suite does not seem to be actively used outside of voyager, I see no reason to keep them in a separate git tree instead of incorporating them in a [git-subtree](https://manpages.debian.org/testing/git-man/git-subtree.1.en.html), keeping the previous history and code attribution but simplifying future changes and repository management?

I'm OK to do the work of merging them and cleaning-up all the dead code; just let me know if it would be OK on the Smithsonian's side? If there is some reason to keep it as-is, I'd like to know :)

**Comment by gjcope** [comment]
No issues on our end.

**Comment by gjcope** [comment]
Submodules have been dropped
